rfcs/models/identity/iin.md

```diff
@@ -25,23 +92,34 @@ The figure below illustrates the architecture of an IIN.
+
+Above the node pool lie agents acting on behalf of trust anchors. These trust anchors can be reputable entity trusted to vouch for DLT network consortiums and their individual units. The trust anchors might be identity validators who assert the physical identity of a particular DID subject, or membership validator asserting the membership of a DID subject in a particular blockckchain network.
+
+In this architecture, the identity validating trust anchors can issue VCs to the participants units such as organizations participating in blockchain networks. Moreover these trust anchors may also issue VCs directly for the Network DIDs, attesting the identity of the network itself.
+
+## Trust Anchors
+
+DID documents are implicitly not tied to any physical identity. Thus after obtaining a Network DID document or a participant DID document, it has to be validated based on some trust basis. For this validation the local network participants may rely on some trust anchors.
 
-Above the node pool lie agents acting on behalf of _IIN stewards_. An steward is a core member of an IIN, and is a reputable entity trusted to vouch for DLT network consortiums and their individual units. (By default, a sample Indy network is configured with global Sovrin stewards, but we can use entities like IBM or Maersk or Walmart for this purpose.)
+These trust anchors are entities that the participants trust and whose identity are already known to them. For example, trust anchors may be well known organizations that validate network identities, just like certificate authorities (CA) in the internet. Which trust anchor to use as trust basis entirely depends on the validator. Any entity with an IIN agent can itself become a trust anchor.
 
-A steward is automatically a _trust anchor_ of an IIN, with privileges to issue DIDs and credentials to other users. Additionally, it has privileges to create more trust anchors. (This is an option that can be exercised for extensibility, when networks wit new governing consortiums are added to the ecosystem. Note that if a new trust anchor is created, it will add another level of indirection between an IIN and a participatinc network.) In the Indy framework, identities creation for trust anchors and other users is initiated through an "invitation" protocol (by a steward and a trust anchor respectively). A trust anchor can also issue verifiable credentials (VCs) to other users (downstream nodes in the hierarchy).
 
-Decentralized identities issued can be _verinyms_ or _pseudonyms_. In the current spec, we will only utilize verinyms, which do not protect the privacy of the identity owner.
+## IIN Agents
 
+DLT networks interact with the IINs through a components called IIN Agent. Each participant unit in a DLT network has have atleast one IIN Agent using which it exposes its identity as well as discovers and verifies foreign network identities. The IIN Agents are detailed [here](./iin-agent.md).
 
+
+<!-- 
```

Comment:
I think we are talking about common trust anchors somewhere else, right? Any reason to retain this commented out section?